presto/create.sql

```diff
@@ -0,0 +1,107 @@
+CREATE TABLE IF NOT EXISTS hits (
+    CounterID INT NOT NULL,
```

Comment:
The schema does NOT match the Thrift message schema:
```
File path:  Downloads/hits.parquet
Created by: parquet-cpp version 1.5.1-SNAPSHOT
Properties: (none)
Schema:
message schema {
  required int64 WatchID;
  required int32 JavaEnable (INTEGER(16,true));
  required binary Title (STRING);
  required int32 GoodEvent (INTEGER(16,true));
  required int64 EventTime;
  required int32 EventDate (INTEGER(16,false));
  required int32 CounterID;
  required int32 ClientIP;
  required int32 RegionID;
  required int64 UserID;
...
```
Can you please double check the schema and fix the DDL here?

Reply:
Fixed now.